docs/adv/troubleshooting/errors.md

```diff
@@ -0,0 +1,328 @@
+Bob`s private key share(s) are imported to a VC that is connected to
+Alice`s Charon node. This is a invalid setup/deployment. 
+Alice`s Charon node should only be connected to Alice`s VC.
+
+Check the partial public key shares of each node inside
+cluster-lock.json and see that matches with the public key inside
+`node(num)/validator_keys/keystore-0.json`.
+
+
+## Teku
+
+### Teku `keystore file` error
+
+Teku sometimes logs an error which looks like
+`Keystore file /opt/charon/validator_keys/keystore-0.json.lock already in use `. 
+This can be solved by deleting the file(s) ending with `.lock` in the folder `.charon/validator_keys`. It is caused by an unsafe shut
+down of Teku (usually by double pressing `Ctrl+C` to shutdown containers faster).
+
+
+## Grafana
+
+### How to fix the Grafana dashboard?
+
+Sometimes, Grafana dashboard doesn`t load any data first time around.
+You can solve this by following the steps below:
+- Click the Wheel Icon > Datasources.
+- Click prometheus.
+- Change the "Access" field from `Server (default)` to `Browser`. Press "Save & Test". It should fail.
+- Change the "Access" field back to `Server (default)` and press "Save & Test". You should be presented with a green success icon saying "Data source is working" and you can return to the dashboard page.
+
+### `N/A` & `No data` in validator info panel
+Can be linked to a Teku keystore issue.
+
+
+## Prometheus
+
+### `Unauthorized: authentication error: invalid token`
+    You can ignore this error unless you have been contacted by the Obol Team
+    with monitoring credentials. In that case, follow [Monitoring your Node](../../run/running/monitoring.md) in our guides. It does not affect cluster performance or prevent the cluster from running.
+
+
+## Docker
+
+### How to fix `permission denied` errors?
+
+Permission denied errors can come up in a variety of manners, particularly
+on Linux and WSL for Windows systems. In the interest of security, the
+charon docker image runs as a non-root user, and this user often does not
+have the permissions to write in the directory you have checked out the code
+to. This can be generally be fixed with some of the following:
+- Running docker commands with `sudo`, if you haven`t [setup docker to be run as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/)
+- Changing the permissions of the `.charon` folder with the commands:
+  - `mkdir .charon` (if it doesn`t already exist);
+  - `sudo chmod -R 666 .charon`.
+
+### I see a lot of errors after running `docker compose up`
+    It`s because both Geth and Lighthouse start syncing and so there`s
+    connectivity issues among the containers. Simply let the containers run for
+    a while. You won`t observe frequent errors when Geth finishes syncing. You
+    can also add a second beacon node endpoint for something like Infura by
+    adding a comma separated API URL to the end of
+    `CHARON_BEACON_NODE_ENDPOINTS` in the docker-compose.yml.
```

Comment:
We use Nethermind nowadays

```suggestion
    It`s because both Nethermind and Lighthouse start syncing and so there's
    connectivity issues among the containers. Simply let the containers run for
    a while. You won't observe frequent errors when Nethermind finishes syncing. You
    can also add a second beacon node endpoint for something like Infura by
    adding a comma separated API URL to the end of
    `CHARON_BEACON_NODE_ENDPOINTS` in the docker-compose.yml.
```